safe_transaction_service/history/indexers/ethereum_indexer.py

```diff
@@ -370,7 +370,7 @@ def process_addresses(
         )
         parameters = self.get_block_numbers_for_search(addresses, current_block_number)
         if parameters is None:
-            return [], current_block_number, True
+            return [], None, current_block_number, True
```

Comment:
If you are returning `None` then return type for the function is `Tuple[Sequence[Any], Optional[int], int, bool]`